launcher/core/launcher.cpp

```diff
@@ -329,6 +330,13 @@ void Launcher::readyRead()
             msg >> d->serverAddress;
             break;
         }
+        case Protocol::ServerLaunchError:
+        {
+            QString reason;
+            msg >> reason;
+            QMessageBox::critical(nullptr, "GammaRay server could not start", "The server could not start: " + reason);
```

Comment:
had that thought too, but must have forgotten about it when it worked so nicely ;)
I'll remove